change/@fluentui-web-components-ec172ca8-23f3-4d01-a969-6504fa67a87a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33023&commitId=aa88013961b5ae2dc58dd97bb3d2a1d01c3046d1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|248| Changed|

</details>


  <details><summary><b>Button</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.Small Icon Only Dark Mode.chromium.png|37| Changed|
|Button.Small With Icon Dark Mode.chromium.png|268| Changed|

</details>


  <details><summary><b>Checkbox</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Circular Checked Dark Mode.chromium.png|10| Changed|
|Checkbox.Checked Dark Mode.chromium.png|14| Changed|
|Checkbox.Default Dark Mode.chromium.png|3| Changed|
|Checkbox.Circular Checked.chromium.png|7| Changed|
|Checkbox.Checked.chromium.png|10| Changed|
|Checkbox.Circular Indeterminate Dark Mode.chromium.png|2| Changed|
|Checkbox.Circular Dark Mode.chromium.png|2| Changed|
|Checkbox.Indeterminate Dark Mode.chromium.png|3| Changed|

</details>


  <details><summary><b>Radio</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio.Checked Dark Mode.chromium.png|5| Changed|
|Radio.Default Dark Mode.chromium.png|2| Changed|
|Radio.Checked.chromium.png|3| Changed|

</details>


  <details><summary><b>RadioGroup</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RadioGroup.Disabled Item Dark Mode.chromium.png|2| Changed|
|RadioGroup.Default Dark Mode.chromium.png|2| Changed|

</details>


  <details><summary><b>Slider</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Size Medium.chromium.png|11| Changed|
|Slider.Slider Steps Vertical.chromium.png|5| Changed|
|Slider.Slider Steps.chromium.png|5| Changed|
|Slider.Default.chromium.png|11| Changed|
|Slider.Vertical.chromium.png|11| Changed|
|Slider.Default Dark Mode.chromium.png|19| Changed|
|Slider.Size Small.chromium.png|6| Changed|

</details>


  <details><summary><b>TextInput</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextInput.Content End Dark Mode.chromium.png|42| Changed|
|TextInput.Appearance Outline Dark Mode.chromium.png|42| Changed|
|TextInput.Appearance Underline Dark Mode.chromium.png|20| Changed|
|TextInput.Content Start Dark Mode.chromium.png|42| Changed|
|TextInput.Content Start End Dark Mode.chromium.png|42| Changed|
|TextInput.Required Dark Mode.chromium.png|42| Changed|
|TextInput.Placeholder Dark Mode.chromium.png|42| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>